src/NuGet.Core/NuGet.PackageManagement/PublicAPI.Unshipped.txt

```diff
@@ -1,19 +1,19 @@
+NuGet.PackageManagement.IPackageRestoreManager.RaiseAssetsFileMissingEventForProjectAsync(bool isAssetsFileMissing) -> void
```

Comment:
Back to unintentional changes again?

Reply:
My bad, I thought this will not affect in anything, but it will create merge conflicts for all open prs when this is merged